test/application_system_test_case.rb

```diff
@@ -3,21 +3,20 @@
 
 class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
   include WebsocketsHelpers
-  # Devise::Test::IntegrationHelpers
+  include Devise::Test::IntegrationHelpers
 
-  # driven_by :selenium, using: :chrome, screen_size: [1400, 1400]
-  driven_by :selenium, using: :headless_chrome do |driver_option|
-    # Without this argument, Chrome cannot be started in Docker
-    driver_option.add_argument('no-sandbox')
-  end
+  driven_by :selenium, using: :chrome, screen_size: [1400, 1400]
+  # driven_by :selenium, using: :headless_chrome do |driver_option|
+  #   # Without this argument, Chrome cannot be started in Docker
+  #   driver_option.add_argument('no-sandbox')
+  # end
```

Comment:
Why this change?

Reply:
This wasn't intended. I will revert this.